appveyor.yml

```diff
@@ -19,15 +19,15 @@ for:
           
   build_script:
     - echo build
-    - dotnet build -f net8.0 -c Debug test/Renci.SshNet.Tests/Renci.SshNet.Tests.csproj
-    - dotnet build -f net8.0 -c Debug test/Renci.SshNet.IntegrationTests/Renci.SshNet.IntegrationTests.csproj
+    - dotnet build -f net9.0 -c Debug test/Renci.SshNet.Tests/Renci.SshNet.Tests.csproj
```

Comment:
Do we want to switch from .NET 8.0 (LTS) to .NET 9.0 (STS, currently still in preview)?
Does AppVeyor even support .NET 9.0?

Reply:
> Do we want to switch from .NET 8.0 (LTS) to .NET 9.0 (STS, currently still in preview)?

Imho it makes sense to test against the latest .NET to catch any regressions. Would even be better to test against all of them, but that's probably not worth the CI overhead. That's up to you.

I didn't want this to be merged while .NET 9 is still in preview, that's why it's still draft.

> Does AppVeyor even support .NET 9.0?

it doesn't. I'm just hoping that it supports it when .NET 9 final is released. If not, the SDK could probably be installed manually somehow.